### PR TITLE
Remove deprecated tools from DevEdition pages.

### PR DESCRIPTION
**Pull request by martinbalfanz**
## Description

Scratchpad and Web Audio Editor have been deprecated and removed from Firefox Developer Tools. We should stop promoting them.

## Issue / Bugzilla link

none

## Testing

tested locally

### CODE REVIEW

File: bedrock/firefox/templates/firefox/developer/includes/features.html [hunk 1]
```diff
@@ -106,28 +106,6 @@ <h3 class="name">{{ _('Memory') }}</h3>
           </p>
         </li>
 
-        <li class="gallery-item feature">
-          <img class="icon" src="{{ static('img/firefox/developer/feature-web-audio.svg') }}" alt="">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/developer/includes/features.html]: Please remove this asset from the repo.

File: bedrock/firefox/templates/firefox/developer/includes/features.html [hunk 2]
```diff
@@ -106,28 +106,6 @@ <h3 class="name">{{ _('Memory') }}</h3>
...
-          <p class="desc">
-            {{ _('The only developer tool for inspecting the Web Audio API.') }}
-            {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
-              <a href="https://developer.mozilla.org/docs/Tools/Web_Audio_Editor" class="features-learn-more" rel="external">{{ _('Learn more about Web Audio Editor') }}</a>
-            {% endif %}
-          </p>
-        </li>
-
-        <li class="gallery-item feature">
-          <img class="icon" src="{{ static('img/firefox/developer/feature-scratchpad.svg') }}" alt="">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/developer/includes/features.html]: Please remove this asset from the repo.

**Comment by alexgibson** [comment]
I still see ScratchPad as a feature in the latest Firefox Nightly, are you sure this is being removed?

Screenshot 2019-07-12 at 09 35 14

Adding a do-not-merge label until we have some official confirmation.

File: bedrock/firefox/templates/firefox/developer/includes/features.html (hunk 1 above)
Comment (martinbalfanz) [reply bedrock/firefox/templates/firefox/developer/includes/features.html]: Oups. Will do!

**Comment by martinbalfanz** [comment]
You're right that Scratchpad is not removed yet. However, we are actively working on it :)

See [bug 1519103](https://bugzilla.mozilla.org/show_bug.cgi?id=1519103) for reference.

**Pull request merged**